test/test.js

```diff
@@ -226,25 +282,6 @@ test('Successfully handles events with no data', t => {
   });
 });
 
-test('Responds with error code (4xx or 5xx) to malformed cloud events', t => {
```

Comment:
Why deleting this?

Reply:
@matejvasek I will elaborate a little more here. The rationale behind this change is rooted in changes that were made in the upstream `cloudevents` module and which recently landed in v3.2.0. That version bump of the module introduced the `[HTTP.binary()](https://github.com/boson-project/faas-js-runtime/pull/68/files#diff-89ae878b1a78765add577244271796f156213cdd0ab6722f641ef96926f38674R54)` API method (and others) that I have made use of in this PR.

But a bigger thing that it did was change how the receiver handles malformed events. A summary of the wider community discussion on why receiving a CloudEvent should provide loose validation is summarized in [this comment](https://github.com/cloudevents/sdk-javascript/pull/328#issuecomment-682059664). 

Because of this change in the upstream validation behavior, an exception is never thrown on our end when receiving an event. In `cloudevents` 3.1.x it would have been and was, and was why this test worked. However, in order to maintain the existing behavior in this module, I would need to add a call to `event.validate()` when receiving them. To me that seems to land more in the user/customer domain. I thought a lot about this while making the upstream changes, and it's been pretty well hashed out in the community. So I think this module should follow the behavior upstream.